docker/api/client.py

```diff
@@ -209,6 +218,20 @@ def __init__(self, base_url=None, version=None,
                 'library.'.format(MINIMUM_DOCKER_API_VERSION)
             )
 
+    def __exit__(self, exc_type, exc_value, traceback):
+        super().__exit__(exc_type, exc_value, traceback)
```

Comment:
We're losing any potential return value here, hence overriding any attempt by the super class to suppress an in-flight exception.